src/v3/src/components/Form/Accordion.tsx

```diff
@@ -10,15 +10,8 @@
  * See the License for the specific language governing permissions and limitations under the License.
  */
 
-import {
-  AccordionDetails,
-  AccordionSummary,
-  AccordionSummaryProps,
-} from '@mui/material';
-import MuiAccordion from '@mui/material/Accordion';
-import { styled } from '@mui/material/styles';
-import { useOdysseyDesignTokens } from '@okta/odyssey-react-mui';
-import { Box, Typography } from '@okta/odyssey-react-mui-legacy';
+import { Accordion as OdyAccordion } from '@okta/odyssey-react-mui';
+import { Box } from '@okta/odyssey-react-mui-legacy';
```

Comment:
nit: I already have a ticket for this so it doesn't need to be fixed here, but we're going to switch all `Box` imports from legacy Odyssey to be from `@mui/material`